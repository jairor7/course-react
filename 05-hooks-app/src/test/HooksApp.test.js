import { shallow } from "enzyme";
import { HooksApp } from "../HooksApp";

describe("Pruebas en componente HooksTest", () => {
  test("Debe de regresar mostrar correctamente el componente <HooksApp />", () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
