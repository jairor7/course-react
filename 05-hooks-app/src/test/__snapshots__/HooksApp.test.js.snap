// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente HooksTest Debe de regresar mostrar correctamente el componente <HooksApp /> 1`] = `
<div>
  <h1>
    Hola mundo!!
  </h1>
</div>
`;
